packages/dapi-grpc/protos/platform/v0/platform.proto

```diff
@@ -270,6 +236,34 @@ message GetIdentityKeysResponse {
   oneof version { GetIdentityKeysResponseV0 v0 = 1; }
 }
 
+message GetIdentitiesKeysRequest {
+
+  message GetIdentitiesKeysRequestV0 {
+    repeated bytes ids = 1;
+    bool prove = 2;
+  }
+
+  oneof version { GetIdentitiesKeysRequestV0 v0 = 1; }
+}
+
+message GetIdentitiesKeysResponse {
+
+  message GetIdentitiesKeysResponseV0 {
+    message Keys { repeated bytes keys_bytes = 1; }
+
+    // Map of identity id to keys
+    message IdentitiesKeys { map<string, Keys> entries = 1; };
```

Comment:
We don't use a map anywhere else and instead give back the values as a vec in the same order it was requested. I'm not super against using a map, I just think that it breaks consistency.